src/lib.rs

```diff
@@ -163,7 +163,7 @@ uniffi::include_scaffolding!("ldk_node");
 
 // Config defaults
 const DEFAULT_STORAGE_DIR_PATH: &str = "/tmp/ldk_node/";
-const DEFAULT_NETWORK: Network = Network::Regtest;
+const DEFAULT_NETWORK: Network = Network::Bitcoin;
```

Comment:
Message on the previous commit needs to be updated to remove the part about updating `DEFAULT_NETWORK`.